pages/events.js

```diff
@@ -1,24 +1,108 @@
-import React from 'react';
+import React from 'react'
+import axios from 'axios'
+import NProgress from 'nprogress'
 
-import Icon from '../components/icon';
-import publicPage from '../hocs/public-page';
+import publicPage from '../hocs/public-page'
+import { futureEventsMeetupURL, pastEventsMeetupURL } from '../common/urls'
+import RowEvent from '../components/row-events'
 
-export default publicPage(() =>
-  <div>
-    <main>
-      <Icon />
-      <h3>Under Construction, Coming Soon!...</h3>
-    </main>
-    <style jsx>{`
-      main {
-        min-height: calc(100vh - 70px);
-        background: #f4f7fb;
-        color: #314159;
-        display: flex;
-        justify-content: center;
-        align-items: center;
-        flex-direction: column;
+export default publicPage(
+  class Events extends React.Component {
+    state = {
+      pastEvents: [],
+      futureEvents: [],
+    }
+
+    async componentDidMount() {
+      NProgress.start()
+      try {
+        const requestPastEvents = await axios.get(pastEventsMeetupURL)
+        NProgress.set(0.4)
+        const requestFutureEvents = await axios.get(futureEventsMeetupURL)
+        await this.setState({
+          pastEvents: requestPastEvents.data,
+          futureEvents: requestFutureEvents.data,
+        })
+        NProgress.done()
```

Comment:
Please move this logic to `getInitialProps` so page can be Server Side Rendered. More Info https://github.com/zeit/next.js/#fetching-data-and-component-lifecycle